infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/SessionManager.java

```diff
@@ -187,40 +192,25 @@ private void updateReplicationParameters(Set<LogReplicationSession> sessionsUnch
      * Create sessions (outgoing and incoming) along with metadata
      *
      */
-    private void createSessions() {
+    private void createSessions(Map<String, ClusterDescriptor> connectionEndpoints) {
```

Comment:
Why do you need this map if you're only using the key?

Reply:
For this PR a map not required. But in the connectionModel PR, the map is used somehow.
I am thinking of leaving it this way and when I rebase connectionModel PR again evaluate if I still need the map of a set would suffice.